.github/workflows/lint-test-sdk.yml

```diff
@@ -5,12 +5,28 @@ on:
     paths:
       - '**/*'
   push:
-    branches: [main]
+    branches: [main, tp/**]
+
+  workflow_dispatch:
```

Comment:
Used to allow dispatching via web API (remotely triggered test run when changes are made to `sdk-test-data`)